packages/form-data/test/form-data.spec.js

```diff
@@ -259,4 +259,37 @@ export const test = (test) => {
       "rename.md"
     );
   });
+
+  test("Should allow passing a form element",  () => {
+    // /** @type {globalThis.HTMLFormElement} */
+    /** @type {any} */
+    const form = {
+      id: "my-form",
+      elements: [
+        {
+          tagName: "INPUT",
+          name: "inside",
+          value: "",
+        },
+        {
+          tagName: "INPUT",
+          name: "outside",
+          value: "",
+          form: "my-form",
+        },
+        {
+          tagName: "INPUT",
+          name: "remember-me",
+          value: "on",
+          checked: true,
+        }
+      ]
+    };
```

Comment:
We need `Object.prototype.toString.call(value) === "[object HTMLFormElement]";` to evaluate correctly.



```js
class FakeForm {
  get [Symbol.toStringTag]() {
    return "HTMLFormElement";
  }
  
  get elements() {
    return [/* ... */]
  }

  get id() {
    return "my-form"
  }
}

let form = new FakeForm();
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag